### PR TITLE
Fixes to utilities used by OSSALifetimeCompletion

### PR DESCRIPTION
**Pull request by meg-gupta**

### CODE REVIEW

**Comment by meg-gupta** [comment]
@swift-ci smoke test

**Comment by meg-gupta** [comment]
@swift-ci smoke test macOS Platform

**Comment by meg-gupta** [comment]
@swift-ci smoke test macOS Platform

File: lib/SIL/Utils/OwnershipUtils.cpp [hunk 1]
```diff
@@ -751,6 +751,30 @@ BorrowedValue BorrowingOperand::getBorrowIntroducingUserResult() {
   llvm_unreachable("covered switch");
 }
 
+SILValue BorrowingOperand::getIntroducingUserResult() {
```
Comment (nate-chandler) [inline lib/SIL/Utils/OwnershipUtils.cpp:754]: Nit: Would `getUserResult` be good?  (Or, what's being introduced?)

**Review by nate-chandler — APPROVED** [review]
LGTM

**Comment by meg-gupta** [comment]
@swift-ci please smoke test and merge

**Comment by meg-gupta** [comment]
@swift-ci please smoke test and merge

**Comment by meg-gupta** [comment]
@swift-ci please smoke test Linux platform

**Pull request merged**